packages/headless/src/features/generated-answer/generated-answer-insight-analytics-actions.ts

```diff
@@ -230,7 +237,9 @@ export const logGeneratedAnswerFeedback = (
 export const logGeneratedAnswerDetailedFeedback = (
   details: string
 ): InsightAction =>
-  makeInsightAnalyticsAction({
+  makeInsightAnalyticsActionFactory(
+    SearchPageEvents.generatedAnswerFeedbackSubmit
```

Comment:
just to make sure here we want to send the same event whether the generatedAnswerFeedback is detailed or not?

Reply:
yes that's what we wanna do.